examples/with-next-ui/pages/api/hello.ts

```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
Let's remove this file as it's unrelated to the example

Reply:
Hello, I have removed this file, thank you so much for your suggestion.